### ISSUE TITLE
Diffirence b/w Training and test Accuracy on FER2013

### ISSUE THREAD

**Issue by NomiMalik0207** [WuJie1010/Facial-Expression-Recognition.Pytorch#23]
Nice work man. I want to know about why model got 90+ accuracy on training and just 70+ on test set?
Is it over fitting? If yes than what should be the possible solution?

**Comment by WuJie1010** [comment]
it may be due to some images in the test set are low-quality. Some images are also confused to people.

**Comment by NomiMalik0207** [comment]
How can i improve the Accuracy?

**Comment by WuJie1010** [comment]
I also get the 70+ accuracy in the test set, I will release the best results if i get it.
